src/graph.js

```diff
@@ -40,7 +40,15 @@ class Graph {
   // Optionally accepts an array of other GraphNodes for the new vertex to be connected to
   // Returns the newly-added vertex
   addVertex(value, edges = []) {
-
+    const node = new GraphNode();
+    node._value = value;
```

Comment:
Don't be accessing the underlying value and edges properties on the node class directly like this. Use the getter and setter methods defined on the `GraphNode` class.